www/syncStatus.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference path="../typings/codePush.d.ts" />
+
+"use strict";
+
+/**
+ * Defines the possible result statuses of the window.codePush.sync operation.
+ */
+enum SyncStatus {
+    /**
+     * The application is up to date.
+     */
+    UP_TO_DATE,
```

Comment:
I called this state "NO_UPDATE_AVAILABLE" on the RN side, but I'm not married to either one :)